invenio_accounts/models.py

```diff
@@ -58,25 +61,35 @@ class User(db.Model, UserMixin):
     id = db.Column(db.Integer, primary_key=True)
 
     email = db.Column(db.String(255), unique=True)
+    """User email."""
 
     password = db.Column(db.String(255))
+    """User password."""
 
     active = db.Column(db.Boolean)
+    """Flag to say if the user is active or not ."""
 
     confirmed_at = db.Column(db.DateTime)
+    """When the user confirm the email address."""
 
     last_login_at = db.Column(db.DateTime)
+    """When the user logged in last time."""
 
     current_login_at = db.Column(db.DateTime)
+    """When user logged in."""
```

Comment:
When the user logged in.
(or you could maybe use 'logged into the current session' for more clarity)